deployments/stacks/dpe-k8s-deployments/main.tf

```diff
@@ -31,14 +31,34 @@ module "trivy-operator" {
 module "airflow" {
   depends_on   = [module.victoria-metrics, module.argo-cd, module.sage-aws-eks-autoscaler]
   source       = "spacelift.io/sagebionetworks/airflow/aws"
-  version      = "0.3.1"
+  version      = "0.3.5"

```

Comment:
Are we self defining the version of these?

Reply:
Yes. When we increment a module in `/modules/main.tf` it pushes out a new version to the spacelift terraform registry:
![image](https://github.com/user-attachments/assets/83c563df-6c0e-4143-a594-c8427a01180a)

We can then install any version of these as well as make breaking changes without affecting anything that might be using older versions. It is slightly overkill since everything is in a single repo (And you could reference it via relative importing). But if we start by following this it should scale out to handle for cases where we have many different deployments using different versions, potentially spread across multiple repos.